app/drivers/kscan/kscan_deghost.c

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
```

Comment:
```suggestion
 * Copyright (c) 2023 The ZMK Contributors
```
(or 2022, depending on when you started writing this)